docs/components/modeler/web-modeler/play-your-process.md

```diff
@@ -107,40 +149,48 @@ Both offer monitoring of a single process instance, its variables and path, inci
 
 ## Limitations and availability
 
-:::note
-Play is being rebuilt and progressively rolled out to more users. This section explains why you might not see the **Play** tab.
-:::
+Play is being rebuilt and progressively rolled out to more users. This section explains why you might not see the **Play** tab, and any additional limitations.
 
-For Camunda 8 SaaS, Play is available to all Web Modeler users with editor or admin permissions within a project.
-Enterprise users need an admin to enable Play by opting in to [alpha features](/components/console/manage-organization/enable-alpha-features.md).
+For more information about terms, refer to our [licensing and terms page](https://legal.camunda.com/licensing-and-other-legal-terms#c8-saas-trial-edition-and-free-tier-edition-terms).
+
+### Camunda 8 SaaS
 
-For Self-Managed, Play is controlled by the **PLAY_ENABLED** flag. It is `true` by default for the Docker distribution for development and `false` by default on the Kubernetes distribution for production use.
+In Camunda 8 SaaS, Play is available to all Web Modeler users with editor or admin permissions within a project.
```

Comment:
This may change depending on the permissions. I think Commenter access is also possible.

You should confirm the org permissions (dev or admin?) and project permissions (commenter, editor, or admin?) that are required to view the Play tab

Reply:
@yT0n1 Are you able to confirm the correct permissions here?

Reply:
For now, yes, though recently, also Commenters have access to Play. We will also limit access to Admin and Developer Organization roles. [But that is not yet implemented](https://github.com/camunda/play/issues/593). So we could leave it for now and revisit this paragraph later?

Reply:
Leaving this as-is for now, and we can update as additional roles are implemented/removed.